sdlib/d/gc/capi.d

```diff
@@ -27,15 +27,23 @@ void* realloc(void* ptr, size_t size) {
  * SDC runtime API.
  */
 void* __sd_gc_alloc(size_t size) {
-	return threadCache.alloc(size, true, false);
+	return threadCache.alloc(size, true, true);
```

Comment:
Do not change this. If you need specific function, put them in the dmd folder. We shouldn't zero everything. You might want ot zero the tail of the allocation like it was done before, but zeroing everything by default is a no go.

Reply:
ok